### PR TITLE
[Mono] Support get_Count and get_AllBitsSet for mini JIT on Arm64

### PR DESCRIPTION
**Pull request by fanyang-mono**
Contributes to #80566

### CODE REVIEW

File: src/mono/mono/mini/simd-intrinsics.c [hunk 1]
```diff
@@ -1944,32 +1944,10 @@ emit_vector64_vector128_t (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 		return NULL;
 #endif
 
-// FIXME: This limitation could be removed once everything here are supported by mini JIT on arm64
+// FIXME: Support Vector64 for mini JIT on arm64
 #ifdef TARGET_ARM64
-	if (!COMPILE_LLVM (cfg)) {
+	if (!COMPILE_LLVM (cfg) && (size != 16))
 		return NULL;
-		if (size != 16)
```
Comment (vargaz) [inline src/mono/mono/mini/simd-intrinsics.c:1951]: Unrelated ?

**Review by vargaz — APPROVED** [review]

**Review by jandupej — APPROVED** [review]
LGTM

File: src/mono/mono/mini/simd-intrinsics.c (hunk 1 above)
Comment (fanyang-mono) [reply src/mono/mono/mini/simd-intrinsics.c:1951]: We haven't tested Vector64 yet. This is a guard to make sure only Vector128 could go through

File: src/mono/mono/mini/simd-intrinsics.c [hunk 2]
```diff
@@ -387,7 +387,7 @@ emit_simd_ins_for_binary_op (MonoCompile *cfg, MonoClass *klass, MonoMethodSigna
 			case SN_Multiply:
 			case SN_op_Multiply: {
 #ifdef TARGET_ARM64
-				if (!COMPILE_LLVM (cfg) && (arg_type == MONO_TYPE_I8 || arg_type == MONO_TYPE_U8))
+				if (!COMPILE_LLVM (cfg) && (arg_type == MONO_TYPE_I8 || arg_type == MONO_TYPE_U8 || MONO_TYPE_I || MONO_TYPE_U))
 					return NULL;
```
[the remark this answers is not in the document]
Comment (fanyang-mono) [reply src/mono/mono/mini/simd-intrinsics.c:391]: right
